Connectors/Revit/Speckle.Connectors.RevitShared/Bindings/SelectionBinding.cs

```diff
@@ -4,13 +4,16 @@
 using Speckle.Connectors.Revit.Plugin;
 using Speckle.Converters.Common;
 using Speckle.Converters.RevitShared.Helpers;
-using Timer = System.Timers.Timer;
 
 namespace Speckle.Connectors.Revit.Bindings;
 
 // POC: we need a base a RevitBaseBinding
-internal sealed class SelectionBinding : RevitBaseBinding, ISelectionBinding
+internal sealed class SelectionBinding : RevitBaseBinding, ISelectionBinding, IDisposable
```

Comment:
@adamhathcock I'm hoping (but not tested) that autofac is smart enough to dispose bindings...